source/2019-12-29-countdown-to-the-new-year-ember-code-snippet.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Countdown to The New Year - Ember Code Snippet
+author: Isaac Lee
+tags: 2019, Addons, DecEmber
+alias: 2019-12-29-countdown-to-the-new-year-ember-code-snippet.md
+responsive: true
+---
+
+**This is the twenty-ninth in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 29
+
+Before I took this assignment, I didn't know anything about [ember-code-snippet](https://emberobserver.com/addons/ember-code-snippet). Now, I'm glad that I did. This addon surfaced in 2015, has received regular updates, and powers many of my favorite addons. (Can you guess how?)
```

Comment:
The personal note in the intro is really cool, imho :tada:

Reply:
Thanks! I felt lucky to have worked on this.